src/psyclone/psyir/nodes/read_only_verify_node.py

```diff
@@ -74,9 +70,7 @@ def __init__(self, ast=None, children=None, parent=None, options=None):
         # If there is no value passed to the constructor, default
         # to the "read_only_verify" prefix.
         my_options["prefix"] = my_options.get("prefix", "read_only_verify")
-        super(ReadOnlyVerifyNode, self).__init__(ast=ast, children=children,
-                                                 parent=parent,
-                                                 options=my_options)
+        super(ReadOnlyVerifyNode, self).__init__(ast=ast, options=my_options)
         self._text_name = "ReadOnlyVerify"
         self._colour = "green"
```

Comment:
Shouldn't this be class attributes? Also see other comments on others constructors as the rest of this method could be deleted if the default_prefix is set on the generic constructor.

Reply:
Yes, good spot. The constructor is gone now.